### PR TITLE
[stats] Fix developer portal charts

### PR DESCRIPTION
**Pull request by didierofrivia**
**What this PR does / why we need it**:

Fixes a bug in the charts displayed on Statistics section on Developer Portal

**Which issue(s) this PR fixes** 

Fixes https://issues.jboss.org/browse/THREESCALE-1616

**Verification steps** 
1. Visit Statistics in Developer Portal
2. Voilà

**Special notes for your reviewer**:
Before:
screen shot 2018-11-30 at 15 52 17

After:
screen shot 2018-11-30 at 15 51 34

### CODE REVIEW

File: lib/developer_portal/app/assets/stylesheets/stats/c3.css [hunk 1]
```diff
@@ -0,0 +1,167 @@
+/*-- Chart --*/
+.c3 svg {
```
Comment (mikz) [inline lib/developer_portal/app/assets/stylesheets/stats/c3.css]: These are our styles?

File: lib/developer_portal/app/assets/stylesheets/stats.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@import 'stats/c3';
```
Comment (mikz) [inline lib/developer_portal/app/assets/stylesheets/stats.scss]: Is this file uploaded to the CMS or loaded from our assets?

File: lib/developer_portal/app/assets/stylesheets/stats/c3.css (hunk 1 above)
Comment (didierofrivia) [reply lib/developer_portal/app/assets/stylesheets/stats/c3.css]: right, should be in a `vendor` folder, no?

Comment (mikz) [reply lib/developer_portal/app/assets/stylesheets/stats/c3.css]: That really depends from where it comes.
Is it from some npm module? Then we should reference that installed module.

Comment (thomasmaas) [reply lib/developer_portal/app/assets/stylesheets/stats/c3.css]: I don't think so, @didierofrivia ?

File: lib/developer_portal/app/assets/stylesheets/stats.scss (hunk 2 above)
Comment (didierofrivia) [reply lib/developer_portal/app/assets/stylesheets/stats.scss]: That one is loaded from our assets stored in cloudfront... However, I've checked the minified version currently stored and it includes it... will keep investigating

**Pull request merged**